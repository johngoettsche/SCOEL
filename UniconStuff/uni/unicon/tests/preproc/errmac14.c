/* test error message: macro paramater must be an identifier */

#define f(3) a
