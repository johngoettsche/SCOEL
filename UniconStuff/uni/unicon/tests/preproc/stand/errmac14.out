pp: File errmac14.c; Line 3: syntax error in #define
